python/version_details/version_details.py

```diff
@@ -520,6 +520,10 @@ def load_data(self, entity):
             self.clear()
             return
 
+        # Ensure project schema will be loaded. The project schema is needed
+        # to reference Version fields in various UI and other elements.
+        shotgun_globals.run_on_schema_loaded(lambda : None, entity.get("project"))
```

Comment:
I'm curious, as well. I believe that run_on_schema_loaded is asynchronous, so this isn't going to stop anything below from running. The lambda will be called, but it's not real clear to me what it's actually going to do.